tests/integration/test_sqlalchemy_integration.py

```diff
@@ -368,3 +379,45 @@ def test_get_table_comment(trino_connection):
         assert actual['text'] is None
     finally:
         metadata.drop_all(engine)
+
+
+@pytest.mark.parametrize('trino_connection', ['memory'], indirect=True)
+@pytest.mark.parametrize('schema', [None, 'test'])
+def test_get_view_names(trino_connection, schema):
+    engine, conn = trino_connection
+    name = schema or engine.dialect._get_default_schema_name(conn)
+    metadata = sqla.MetaData(schema=name)
+
+    if not engine.dialect.has_schema(conn, name):
+        engine.execute(sqla.schema.CreateSchema(name))
+
+    try:
+        create_view(
+            'my_view',
+            sqla.select(
+                [
+                    sqla.Table(
+                        'my_table',
+                        metadata,
+                        sqla.Column('id', sqla.Integer),
+                    ),
+                ],
+            ),
+            metadata,
+            cascade_on_drop=False,
+        )
+
+        metadata.create_all(engine)
+        assert sqla.inspect(engine).get_view_names(schema) == ['my_view']
+    finally:
+        metadata.drop_all(engine)
+
+
+@patch('trino.sqlalchemy.dialect.TrinoDialect._get_default_schema_name')
```

Comment:
If you remove the default schema from the fixture you can rid yourself of this `@patch`